src/main/java/com/ontotext/kafka/service/GraphDBService.java

```diff
@@ -84,6 +84,8 @@ private SinkRecordsProcessor fetchProcessor(String transactionType, String rdfFo
 						ValueUtil.getRDFFormat(rdfFormat), batchSize, timeoutCommitMs);
 			case SMART_UPDATE:
 			case REPLACE_GRAPH:
```

Comment:
You can change the order here, as SMART_UPDATE is not yet implemented and when written like that, it would use the ReplaceGraphProcessor